### ISSUE TITLE
Option to output evaluation metric scores

### ISSUE THREAD

**Issue by xiaohan2012** [keelm/SeCo-MLC#4]
Hi,

Is there any support of saving the evaluation metric scores (e.g., subset accuracy, micro F1) into the output file?

Han

**Comment by xiaohan2012** [comment]
Hi,

I managed to find the `-evaluate`switch: after adding `-evaluate true` to the command, the program seems to do some data split and  evaluation.

But still I have two questions:

- what is the program doing? split the training data into training and validation sets, doing hyperparameter tuning and report the test scores?
- why do the same metric names appear twice in the result file? e.g., in my running, `Hamming Loss` appears twice, giving `0.0013028103480364787` and `0.0014398848092152625`, respectively. Do they correspond to validation and test scores?

Thanks,
Han

**Comment by keelm** [comment]
Hi,
which branch are you using? Can you tell me the command line you use and maybe attach the output. It's time ago I worked with that code...
Cheers
